ydb/library/yql/yt/native/plugin.cpp

```diff
@@ -465,7 +465,17 @@ class TYqlPlugin
             ActiveQueriesProgress_[queryId].Program = program;
         }
 
-        program->AddCredentials({{"default_yt", NYql::TCredential("yt", "", token)}});
+        TVector<std::pair<TString, NYql::TCredential>> credentials;
+        const auto credentialsMap = NodeFromYsonString(credentialsStr.ToString()).AsMap();
+        credentials.reserve(credentialsMap.size() + credentials.size());
```

Comment:
Why do we write ``` + credentials.size()``` here?

Reply:
I just forgot to fix this place after a draft.